### PR TITLE
Remove broken link in verifying smart contracts developer docs

### PR DESCRIPTION
**Pull request by MahendraBishnoi29**
## Description
fix broken for [this](https://ethereum.org/en/developers/docs/smart-contracts/verifying/) page

### CODE REVIEW

File: src/content/developers/docs/smart-contracts/verifying/index.md [hunk 1]
```diff
@@ -104,5 +104,5 @@ Using the Tenderly Hardhat plugin allows for more control over the verification
 
 ## Further reading {#further-reading}
 
-- [How to verify the source code of Ethereum smart contract](https://developpaper.com/how-to-verify-the-source-code-of-ethereum-smart-contract/)
+- [How to verify the source code of Ethereum smart contract](https://ethereum.org/en/developers/docs/smart-contracts/verifying/)
```
Comment (minimalsm) [inline src/content/developers/docs/smart-contracts/verifying/index.md:107]: Use relative links when linking internally: https://ethereum.org/en/contributing/style-guide/content-standardization/#internal-links

```suggestion
- [How to verify the source code of Ethereum smart contract](/developers/docs/smart-contracts/verifying/)
```

**Review by minimalsm — CHANGES_REQUESTED** [review]
Hey @MahendraBishnoi29, thanks for the PR ❤️ 

One small change and we can get this merged 🙏

File: src/content/developers/docs/smart-contracts/verifying/index.md (hunk 1 above)
Comment (minimalsm) [reply src/content/developers/docs/smart-contracts/verifying/index.md:107]: Actually, this link references the page it is on.... if this link is dead, we should just remove it 👍

Comment (MahendraBishnoi29) [reply src/content/developers/docs/smart-contracts/verifying/index.md]: > Actually, this link references the page it is on.... if this link is dead, we should just remove it 👍

understood 😁

File: src/content/developers/docs/smart-contracts/verifying/index.md [hunk 2]
```diff
@@ -104,5 +104,5 @@ Using the Tenderly Hardhat plugin allows for more control over the verification
 
 ## Further reading {#further-reading}
 
-- [How to verify the source code of Ethereum smart contract](https://developpaper.com/how-to-verify-the-source-code-of-ethereum-smart-contract/)
+- [How to verify the source code of Ethereum smart contract](/developers/docs/smart-contracts/verifying/)
```
Comment (minimalsm) [inline src/content/developers/docs/smart-contracts/verifying/index.md:107]:
```suggestion
```

**Review by minimalsm — APPROVED** [review]
LGTM, thanks @MahendraBishnoi29 ❤️

**Comment by minimalsm** [comment]
@all-contributors please add @MahendraBishnoi29 for bugs

**Comment by MahendraBishnoi29** [comment]
> @all-contributors please add @MahendraBishnoi29 for bugs

haha i guess I'm already there since I'm an active contributor here @minimalsm 😁

**Comment by MahendraBishnoi29** [comment]
> Hey @MahendraBishnoi29, thanks for the PR ❤️
> 
> One small change and we can get this merged 🙏

the link is still broken [check](https://ethereum.org/en/developers/docs/smart-contracts/verifying/#further-reading) here

**Pull request merged**